### PR TITLE
Google GRPC initial commit (Core, Stub, OkHttp, Protobuf.Lite)

### PR DESCRIPTION
**Pull request by moljac**

### CODE REVIEW

File: XPlat/Google.Grpc/build.cake [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+	.IsDependentOn("libs")
+	.Does
+	(
+		() =>
+		{
+			EnsureDirectoryExists("./output");
+
+			MSBuild
+			(
+				"./source/Xamarin.Protobuf.Lite.Bindings.XamarinAndroid/Xamarin.Protobuf.Lite.Bindings.XamarinAndroid.csproj",
```
Comment (newky2k) [inline XPlat/Google.Grpc/build.cake]: This path is not the correct path for the project

./source/Xamarin.Grpc.Protobuf.Lite.Bindings.XamarinAndroid/Xamarin.Grpc.Protobuf.Lite.Bindings.XamarinAndroid.csproj

File: XPlat/Google.Grpc/source/Xamarin.Grpc.Core.Bindings.XamarinAndroid/Xamarin.Grpc.Core.Bindings.XamarinAndroid.csproj [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        <Title>Xamarin.Grpc.Core</Title>
+        <Summary>
+            Google GRPC Core bindings for Xamarin
+        </Summary>
+        <Description>
+            Bindings for Google's GRPC Core package (Google Play Services dependency)
+        </Description>
+        <Owners>Microsoft Corporation</Owners>
+        <Authors>Microsoft / Xamarin</Authors>
+        <Copyright>Copyright 2018</Copyright>
```
Comment (newky2k) [inline XPlat/Google.Grpc/source/Xamarin.Grpc.Core.Bindings.XamarinAndroid/Xamarin.Grpc.Core.Bindings.XamarinAndroid.csproj]: Copyright is incorrect

File: XPlat/Google.Grpc/source/Xamarin.Grpc.Core.Bindings.XamarinAndroid/Xamarin.Grpc.Core.Bindings.XamarinAndroid.csproj [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        <PackageLicenseUrl>
+        https://github.com/grpc/grpc/blob/master/LICENSE
+        </PackageLicenseUrl>
+        <PackageProjectUrl>
+        https://github.com/grpc/grpc
```
Comment (newky2k) [inline XPlat/Google.Grpc/source/Xamarin.Grpc.Core.Bindings.XamarinAndroid/Xamarin.Grpc.Core.Bindings.XamarinAndroid.csproj]: links should be fwlinks

File: XPlat/Google.Grpc/source/Xamarin.Grpc.Core.Bindings.XamarinAndroid/Xamarin.Grpc.Core.Bindings.XamarinAndroid.csproj [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        </PackageProjectUrl>
+        <RepositoryUrl>
+        https://github.com/grpc/grpc
```
Comment (newky2k) [inline XPlat/Google.Grpc/source/Xamarin.Grpc.Core.Bindings.XamarinAndroid/Xamarin.Grpc.Core.Bindings.XamarinAndroid.csproj]: links should be fwlinks

File: XPlat/Google.Grpc/source/Xamarin.Grpc.Core.Bindings.XamarinAndroid/Xamarin.Grpc.Core.Bindings.XamarinAndroid.csproj [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+            Google GRPC Core bindings for Xamarin
+        </Summary>
+        <Description>
+            Bindings for Google's GRPC Core package (Google Play Services dependency)
+        </Description>
+        <Owners>Microsoft Corporation</Owners>
+        <Authors>Microsoft / Xamarin</Authors>
+        <Copyright>Copyright 2018</Copyright>
+        <PackageLicenseUrl>
+        https://github.com/grpc/grpc/blob/master/LICENSE
```
Comment (newky2k) [inline XPlat/Google.Grpc/source/Xamarin.Grpc.Core.Bindings.XamarinAndroid/Xamarin.Grpc.Core.Bindings.XamarinAndroid.csproj]: links should be fwlinks

File: XPlat/Google.Grpc/source/Xamarin.Grpc.Core.Bindings.XamarinAndroid/Xamarin.Grpc.Core.Bindings.XamarinAndroid.csproj [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+        <PackageId>Xamarin.Grpc.Core</PackageId>
+        <PackageVersion>1.14.0</PackageVersion>
+        <Title>Xamarin.Grpc.Core</Title>
+        <Summary>
+            Google GRPC Core bindings for Xamarin
+        </Summary>
+        <Description>
+            Bindings for Google's GRPC Core package (Google Play Services dependency)
+        </Description>
+        <Owners>Microsoft Corporation</Owners>
```
Comment (newky2k) [inline XPlat/Google.Grpc/source/Xamarin.Grpc.Core.Bindings.XamarinAndroid/Xamarin.Grpc.Core.Bindings.XamarinAndroid.csproj]: Should just be microsoft

File: XPlat/Google.Grpc/source/Xamarin.Grpc.Core.Bindings.XamarinAndroid/Xamarin.Grpc.Core.Bindings.XamarinAndroid.csproj [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+        <Authors>Microsoft / Xamarin</Authors>
```
Comment (newky2k) [inline XPlat/Google.Grpc/source/Xamarin.Grpc.Core.Bindings.XamarinAndroid/Xamarin.Grpc.Core.Bindings.XamarinAndroid.csproj]: should just be microsoft

**Review by newky2k — CHANGES_REQUESTED** [review]
Found some issues with the package meta data, needs to be fixed in all of the packages, but nuget target fails as the path to the csproj is incorrect

**Comment by moljac** [comment]
review remarks fixes done!

**Review by newky2k — APPROVED** [review]

**Pull request merged**